public/Aryan Suri/Sidebar/sidebar.js

```diff
@@ -261,117 +350,68 @@ async function Sidebar() {
 	}
 
 
-	function getData(pro) {
+	function getData(pro, tabs) {
 
 
 		return {
 
 			"open": `<button id="custom_open"  >☰</button>`,
-			"headernr": pro ? `<div id="sbHeader" class="top-bar" style="">
-			<div class="top-bar-unit" >
-				<h5 id="openContents">Contents</h5>
-			</div>
-			
-			<div class="top-bar-unit">
-				<h5 id="openControl">Control</h5>
-			</div>  
-			
-			<div class="top-bar-unit">
-				<h5 id="openUsage"  class="">Usage</h5>
-			</div>
-	
-			<div class="top-bar-unit">
-				<h5 id="openDevelopers" class="">Developers</h5>
-			</div>
-	
-		   
-			</div>` : `<div id="sbHeader" class="top-bar" style="">
-			<div class="top-bar-unit" >
-				<h5 id="openContents">Contents</h5>
-			</div>
-		
-			<div class="top-bar-unit">
-				<h5 id="openControl">Control</h5>
-			</div>  
-			
-			<div class="top-bar-unit">
-				<h5 id="openUsage"  class="">Usage</h5>
-			</div>
-	
-			</div>`,
-			"header": pro ? `<div id="sbHeader" class="top-bar" style="">
-        <div class="top-bar-unit" >
-            <h5 id="openContents">Contents</h5>
-        </div>
-    
-        <div class="top-bar-unit">
-            <h5 id="openResources" class="">Resources</h5>
-        </div>
-
-        <div class="top-bar-unit">
-            <h5 id="openControl">Control</h5>
-        </div>  
-        
-        <div class="top-bar-unit">
-            <h5 id="openUsage"  class="">Usage</h5>
-        </div>
-
-        <div class="top-bar-unit">
-            <h5 id="openDevelopers" class="">Developers</h5>
-        </div>
-
-       
-        </div>` : ` <div id="sbHeader" class="top-bar" style="">
-        <div class="top-bar-unit" >
-            <h5 id="openContents">Contents</h5>
-        </div>
-    
-        <div class="top-bar-unit">
-            <h5 id="openResources" class="">Resources</h5>
-        </div>
 
-        <div class="top-bar-unit">
-            <h5 id="openControl">Control</h5>
-        </div>  
-        
-        <div class="top-bar-unit">
-            <h5 id="openUsage"  class="">Usage</h5>
-        </div>
-       
-        </div> `,
+			"header": pro ? `<div id="sbHeader" class="sbHeader" style="">
+        <div id="openContents"  class="top-tabs">
+			<h5  >Contents</h5>
+		</div>
+		<div  id="openResources" class="top-tabs">
+            <h5 class="">Resources</h5>
+		</div>
+		<div id="openControl"  class="top-tabs">
+			<h5 >Readability</h5>
+		</div>
+		<div  id="openUsage"  class="top-tabs">
+            <h5 class="">Application</h5>
+		</div>
+
+		<div id="openLibreverse"  class="top-tabs">
+            <h5  class="">LibreVerse</h5>
```

Comment:
Nonblocking, but I would recommend running this through a formatter to clean up these h5 tags.